Dockerfile

```diff
@@ -9,6 +9,12 @@ COPY package.json package-lock.json ./
 # Install Node Packages
 RUN npm install
 
+# Copying self-signed certs
+COPY certs/ certs/
+
+# Generate certs
+RUN cd certs && ./certgen.sh && cd ..
+
```

Comment:
Furthermore, does `node:14.13.1` have openssl installed by default? It is necessary to run the script.

Reply:
I ran the server with Docker and it worked as expected, so I assume it has :P